python/metrics/issue_comment_similartiy.py

```diff
@@ -0,0 +1,141 @@
+﻿import string
+from tqdm import tqdm
+import pandas as pd
+import db.clickhouse as clickhouse
+import numpy as np
+from nltk.corpus import stopwords
+from sklearn.feature_extraction.text import CountVectorizer
+from scipy.linalg import norm
+from sklearn.feature_extraction.text import TfidfVectorizer
+
+def getSelectedActors(config):
+    """
+    TODO: get Selected Acotrs
+    """
+    sql = 'SELECT DISTINCT(actor_id) FROM opensource.gh_events')
```

Comment:
I think we can add a limit clause such as `limit 10`

Reply:
The function aims to find all users if have no selecting choice.

Reply:
`DISTINCT` selection from the whole table may lead to a OOM exception since the ClickHouse only has 128GB memory, we should definitely avoid this kind of SQL.